### ISSUE TITLE
Project 4 Peer Review

### ISSUE THREAD

**Issue by siddharthpathak** [airavata-courses/Schrodinger-s-cat#101]
Where can I try/test the application out?

**Comment by pswargam** [comment]
Hello Siddarth ,

You can access the product at 129.114.16.182:31012/

**Comment by nawazkh** [comment]
Hello Siddharth, 

Our application is up and running if you want to test it. 
The link is mentioned in the assignment 4. 

Also, you can check our dashboard for Kubernetes installation. You will need the admin token for login.
We will get it to you in personal chat once you ping us.


All the documentation and wiki will be done soon.

Thanks,
nawaz

**Comment by siddharthpathak** [comment]
I was able to sign-up, but I am not able to log in. The browser network tab shows request failed/timed out. Nice work with the sign-up email. 
Yes can you please provide me the login token for Kubernetes cluster.

**Comment by nawazkh** [comment]
I guess the issue is because we have not put any check on using special characters on the password field. 
I would recommend creating an alphanumeric password for a new account.

For the token, will be sending you in a personal chat

**Comment by siddharthpathak** [comment]
Was able to login after creating a new account. UI has improved a lot as compared to previous milestones. 
Tested Jenkins build by committing on the branch, and the pod images got updated correctly. 
Good work with wiki and diagrams. Liked that you were able to deploy Kubernetes dashboard on public IP. We faced issues in making it accessible outside the cluster.
Also, I am curious about how Kubernetes compares with DockerSwarm which you had in previous milestone.
Overall good work!
